tapdb/sqlc/migrations/000011_transfers.up.sql

```diff
@@ -0,0 +1,17 @@
+DROP TABLE IF EXISTS receiver_proof_transfer_attempts;
```

Comment:
How will this affect clients that are updating mid send/recv?

Reply:
I think that in the worst case they'll just skip a delay in their interaction with the courier service.